src/org/parosproxy/paros/core/scanner/HostProcess.java

```diff
@@ -757,6 +765,17 @@ public void alertFound(Alert alert) {
             pluginStats.incAlertCount();
         }
         alertCount++;
+        
+        ExtensionAlert extensionAlertRef = Control.getSingleton().getExtensionLoader().getExtension(ExtensionAlert.class);

```

Comment:
We need to do this check before notifying that the alert was found, in daemon it's not working properly since the alert is actually raised before this check (so it's never new).